athf/mcp/server.py

```
@@ -51,10 +51,13 @@ def create_server(workspace_path: Optional[str] = None) -> "FastMCP":  # type: i
     """
     try:
         from mcp.server.fastmcp import FastMCP
-    except ImportError:
+    except ImportError as exc:
         raise ImportError(
-            "MCP dependencies not installed. Install with: pip install 'athf[mcp]'"
-        ) from None
+            "Could not import mcp.server.fastmcp.FastMCP ({}). This usually means "
+            "either the mcp package is not installed, or an incompatible version "
+            "is installed: mcp 2.0.0 removed FastMCP. Install a supported version "
+            "with: pip install 'athf[mcp]' (which pins mcp[cli]>=1.0.0,<2.0.0).".format(exc)
```

Ruff (0.16.0)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)